react/admin/ReturnDetails/components/ReturnLabel/ReturnLabel.tsx

```diff
@@ -0,0 +1,194 @@
+
+  const { data: installedApp } = useQuery(GET_APP, {
+    variables: {
+      slug: 'vtex.easypost',
+    },
+  })
+
+  useEffect(() => {
+    if (!installedApp?.app) return
+
+    const { app } = installedApp
+
+    const appSettings = JSON.parse(app.settings)
+    const { street1, street2, city, state, zip, country, name, phone } =
+      appSettings
+
+    setReturnAddress({
+      street1,
+      street2,
+      city,
+      state,
+      zip,
+      country,
+      name,
+      phone,
+    })
+
+    setDisplayButton(true)
+  }, [installedApp])
+
+  useEffect(() => {
+    if (!data) return
+
+    const {
+      status,
+      pickupReturnData: { returnLabel },
+    } = data.returnRequestDetails
+
+    if (status === 'processing') {
+      setDisableCreateLabel(false)
+
+      if (returnLabel) {
+        setLabelUrl(returnLabel)
+        setDisbleLabelUrl(false)
+        setDisableCreateLabel(true)
+      }
+    }
+  }, [data])
+
+  const handleToggleModal = () => {
+    setIsModalOpen(!isModalOpen)
+  }
+
+  const handleCancelation = () => {
+    setIsModalOpen(false)
+  }
+
+  const [, { loading: creatingLabel }] = useMutation(CREATE_LABEL)
+  const [sendLabel, { loading: sendingEmail }] = useMutation(SEND_LABEL)
+
+  const handleConfirmation = async () => {
+    // temp labelUrl [DELETE when we have the client Key from easypost]
+    const createdLabelUrl =
+      'https://assets.easypost.com/assets/images/usps-international-label.c7c603e0b25b12e4489a8c75db0d34b8.png'
+
+    try {
+      // Send mutation to EasyPost
+      // const createdLabelUrl = await createLabel({
+      //   variables: {
+      //     ...returnAddress,
+      //   },
+      // })
+
+      await sendLabel({
+        variables: {
+          requestId: data?.returnRequestDetails.id,
+          labelUrl: createdLabelUrl,
+        },
+      })
+
+      openAlert(
+        'success',
+        <FormattedMessage id="admin/return-app.return-request-details.return-label.alert.success" />
+      )
+    } catch (error) {
+      openAlert(
+        'error',
+        <FormattedMessage id="admin/return-app.return-request-details.return-label.alert.error" />
+      )
+    }
+
+    setIsModalOpen(false)
+  }
+
+  const handleOpenLabelUrl = () => {
+    window.open(labelUrl, '_blank')
+  }
+
+  if (!displayButton) return null
+
+  return (
+    <div className="mb4">
+      {displayButton && (
+        <div className="flex">
+          <div className="mr4">
+            <ButtonPlain
+              className="mr4"
+              disabled={disableCreateLabel}
+              onClick={handleToggleModal}
+            >
+              <FormattedMessage id="admin/return-app.return-request-details.return-label.create-return-label" />
+            </ButtonPlain>
+          </div>
+          <ButtonPlain disabled={disableLabelUrl} onClick={handleOpenLabelUrl}>
+            <FormattedMessage id="admin/return-app.return-request-details.return-label.see-return-label" />
+          </ButtonPlain>
```

Comment:
I think we should show one or the other button, but not the two of them at the same time.